lib/minidump.js

```diff
@@ -61,6 +61,7 @@ module.exports.walkStack = function (minidump, symbolPaths, callback) {
   }
 
   var args = [minidump].concat(symbolPaths, globalSymbolPaths)
+  args = commandArgs ? [commandArgs].concat(args) : args;
```

Comment:
This project follows JavaScript Standard Style (http://standardjs.com) which does not use `;` at the end of statement, can your remove it?